tools/infer/utility.py

```diff
@@ -220,7 +220,7 @@ def create_predictor(args, mode, logger):
                 logger.warning(
                     "GPU is not found in current device by nvidia-smi. Please check your device or ignore it if run on jetson."
                 )
-            config.enable_use_gpu(args.gpu_mem, args.gpu_id)
+            config.enable_use_gpu(args.gpu_mem, 0)
```

Comment:
please explain the reason for making this change. it seems irrelevant to the purpose of the PR.

Reply:
sorry, this change is wrong. Now have re-PR: https://github.com/PaddlePaddle/PaddleOCR/pull/10421